CVARS.rst

```diff
@@ -45,6 +45,23 @@ New and Modified Cvars
    | ``fs_game cvar``
    | ``fs_forcegame cvar``
 
+..
+
+:Name: fs_assetsPathJKA
+:Values: Foldername
+:Default: "" (Not set on portable); autodetected for non-portable
+:Description:
+   Sets the path to load JKA assets from.
+
+..
+
+:Name: fs_noJKA
```

Comment:
I think this is pretty important cvar. Loading all these assets may affect a lot of things, especially where mods use trap_FS_GetFiles(). It may even cause some mods to break/crash (eg. run out of some file array size). I would add it to our menus.

Not a fan of reverse logic ("noJKA") :P how about fs_jka, fs_loadjka, fs_jediacademy? If you want it to leave it as fs_nojka then I don't mind that much.

Reply:
Renamed it to `fs_loadjka`.

I agree that we should add it to the menus, but I was intending to look into it after it has been merged (really hoping for someone else to do it if I am being honest).